core/core-impl/src/main/java/ai/timefold/solver/core/impl/score/stream/bi/ObjectCalculatorBiCollector.java

```diff
@@ -10,9 +10,9 @@
 
 abstract sealed class ObjectCalculatorBiCollector<A, B, Input_, Output_, Calculator_ extends ObjectCalculator<Input_, Output_>>
         implements BiConstraintCollector<A, B, Calculator_, Output_>
-        permits AverageReferenceBiCollector, CountDistinctIntBiCollector, CountDistinctLongBiCollector,
-        SumReferenceBiCollector {
-    private final BiFunction<? super A, ? super B, ? extends Input_> mapper;
+        permits AverageReferenceBiCollector, ConsecutiveSequencesBiConstraintCollector, CountDistinctIntBiCollector,
+        CountDistinctLongBiCollector, SumReferenceBiCollector {
+    protected final BiFunction<? super A, ? super B, ? extends Input_> mapper;
```

Comment:
Can remain private; call `super.equals` in subclasses' `equals` method.

Reply:
See above for explanation.